creativecloud/scripts/utils.js

```diff
@@ -38,3 +38,49 @@ export const [setLibs, getLibs] = (() => {
     }, () => libs,
   ];
 })();
+
+function getDecorateAreaFn() {
+  let lcpImgSet = false;
+
+  // Load LCP image immediately
+  const eagerLoad = (lcpImg) => {
+    lcpImg?.setAttribute('loading', 'eager');
+    lcpImg?.setAttribute('fetchpriority', 'high');
+    if (lcpImg) lcpImgSet = true;
+  };
+
+  async function loadLCPImage(area = document, { fragmentLink = null } = {}) {
+    const firstBlock = area.querySelector('body > main > div > div');
+    let fgDivs = null;
+    switch (true) {
+      case firstBlock?.classList.contains('changebg'): {
+        firstBlock.querySelector(':scope > div:nth-child(1)').querySelectorAll('img').forEach(eagerLoad);
```

Comment:
will 'img' always be there, or do we need '?' as:
`firstBlock.querySelector(':scope > div:nth-child(1)').querySelectorAll('img')?.forEach` just to be sure?

Reply:
ah i see that you just moved this code piece, but still wondering if we need to add those checks or good as is..

Reply:
Good question but as you mentioned, this code has not been touched and merely moved.